src/php/Runtime/Runtime.php

```diff
@@ -85,6 +95,9 @@ public function exceptionHandler(Throwable $exception): void
      */
     public function loadNs(string $ns): bool
     {
+        return true;
```

Comment:
😱😜

Reply:
I have to check a few edge cases first before removing this completely :)

Reply:
I removed all calls to `loadNs` is this project. However, the Phel Composer Plugin still call this method, so I can not fully remove it.